lib/tile_bag.rb

```diff
@@ -0,0 +1,45 @@
+module Scrabble
+  class TileBag
+    attr_reader :letter_quantity
+
+    def initialize
+      @letter_quantity = {A:9, B:2, C:2, D:4, E:12, F:2, G:3, H:2, I:9, J:1, K:1,
+         L:4, M:2, N:6, O:8, P:2, Q:1, R:6, S:4, T:6, U:4, V:2, W:2, X:1, Y:2, Z:1}
```

Comment:
This is a good, readable data structure for storing the starting tiles. However, it means you need a really complicated loop below in `draw_tiles`. It might be cleaner to walk through this hash here and add the right amount of each letter to a big array, then draw from that instead.